packages/vuelidate/src/core.js

```diff
@@ -284,6 +289,11 @@ function createValidationResults (rules, state, key, parentKey, resultsCache, pa
     })
   )
 
+  result.$errors = computed(() => result.$dirty.value
```

Comment:
Wont this populate, only if the current level is dirty? I am not entirely sure what outcome we want.

Reply:
Good question. Let’s get back to it later on.